adapters/nanointeractive/usersync_test.go

```diff
@@ -0,0 +1,26 @@
+package nanointeractive
+
+import (
+	"testing"
+	"text/template"
+
+	"github.com/prebid/prebid-server/privacy"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewNanoInteractiveSyncer(t *testing.T) {
+	syncURL := "//ad.audiencemanager.de/hbs/cookieSync/{{.GDPR}}"
```

Comment:
This is usually a copy of the `setDefaultUsersync` and is intended to ensure there are no typos.